### ISSUE TITLE
Accessibility of carousel images

### ISSUE THREAD

**Issue by aferreira-deo** [ivylaboratory/angular-carousel#143]
Hello ivylaboratory team, it seems there is an accessibility issue for the images when you need to use a screen reader. In order for the images to have some description for screen readers to comply with ([WCAG rule 1.1.1](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content)) the images need to have an alt attribute that can be configured to either describe the picture or have an empty string in case the image is not meaningful and it should be omitted from the screen reader elements.

Would it be possible to extend the Image and Images interfaces with an alt attribute, you could put it as optional if you don't want a breaking change. Also if the attribute is empty you could default to empty string '' so the screen reader ignores the image.

I understand we can have control when we use content projection, but when we just pass the [images] input we are unable to provide alt attributes.
